source/includes/environment_config/_45-patch.md.erb

```diff
@@ -139,6 +162,25 @@ describe_sub_object 'The environments agent-update operation attributes', json:
 end
 %>
 
+<%
+json = {
+    add: [
+      {
+        :name => 'GO_SERVER_URL',
+        :value => 'https://ci.example.com/go'
+      }
+    ],
+    remove: ['URL']
+  }
+%>
+
+<%=
+describe_sub_object 'The environments environment-variable-update operation attributes', json: json, html_id: 'the-environments-environment-variables-update-operation' do
+  add              'Array',   'The list of environment variables which needs to be added to the specified environment'
```

Comment:
Link to the env var description in pipeline config. People may want to specify `secure` and stuff right?